apps/src/blockly/addons/contextMenu.ts

```diff
@@ -145,7 +145,11 @@ const registerShadow = function (weight: number) {
   const shadowOption = {
     displayText: () => 'Make Shadow',
     preconditionFn: function (scope: GoogleBlockly.ContextMenuRegistry.Scope) {
-      if (Blockly.isStartMode && scope.block && canBeShadow(scope.block)) {
+      if (
+        (Blockly.isStartMode || Blockly.isToolboxMode) &&
```

Comment:
This change, and the one below it, allow levelbuilders to toggle blocks to shadow blocks and back.